libraries/toolbar/src/main/java/com/trendyol/uicomponents/toolbar/ToolbarViewState.kt

```diff
@@ -19,7 +19,14 @@ data class ToolbarViewState(
     @StyleRes val middleTextAppearance: Int = R.style.Trendyol_UIComponents_Toolbar_Text_Title,
     @StyleRes val upperRightTextAppearance: Int = R.style.Trendyol_UIComponents_Toolbar_Text_UpperAction,
     @StyleRes val lowerRightTextAppearance: Int = R.style.Trendyol_UIComponents_Toolbar_Text_LowerAction,
-    @DrawableRes val toolbarBackground: Int = android.R.color.white
+    @DrawableRes val toolbarBackground: Int = android.R.color.white,
```

Comment:
is this parameter used?

Reply:
It is used to set background image resource drawable in toolbar layout